data/analytics/attributes.yml

```diff
@@ -22,6 +30,22 @@
   type: string
   redact: false
 
+- name: cookie_banner
+  description: Used in the page_view object to record the presence of the cookie banner "You have accepted cookies" element on the page
+  value: true/undefined depending on the presence of data-ga4-cookie-banner on the page
+  example: "true"
+  required: no
+  type: string (boolean)
+  redact: false
+
+- name: devolved_nations_banner
+  description: Used in the page_view object to record the countries listed in a devolved nations banner on the page
+  value: The value of data-ga4-devolved-nations-banner on the page
+  example: "England, Scotland, Wales"
```

Comment:
Just to check, does the actual data for this example contain spaces between the countries?

Reply:
@andysellick Yep 👍